docs/griptape-framework/drivers/prompt-drivers.md

```diff
@@ -416,7 +431,8 @@ from griptape.config import StructureConfig
 agent = Agent(
     config=StructureConfig(
         prompt_driver=AmazonSageMakerPromptDriver(
-            model=os.environ["SAGEMAKER_LLAMA_ENDPOINT_NAME"],
+            endpoint=os.environ["SAGEMAKER_LLAMA_3_INSTRUCT_ENDPOINT_NAME"],
+            model=os.environ["SAGEMAKER_LLAMA_3_INSTRUCT_INFERENCE_COMPONENT_NAME"],
```

Comment:
Needs to be updated in https://github.com/griptape-ai/griptape/blob/81ec3b637ae9c924a80ecdfdfb472f760ef99ee1/.github/workflows/docs-integration-tests.yml#L80